src/ddmd/parse.d

```diff
@@ -5077,7 +5077,7 @@ final class Parser(AST) : Lexer
 
                 nextToken();
                 //if (token.value == TOKsemicolon)
-                //    error("use '{ }' for an empty statement, not a ';'");
+                //    error("use `{ }` for an empty statement, not `;`");
```

Comment:
As you know, I've long argued for smaller, more focused commits. But this is in response to PRs with hundreds of lines of unrelated changes mixed up together, in order to make reviewing easier, and isolating regressions easier. This is fixing a grammatical error in a line that is already being modified. Putting this in a separate commit does not make it easier to review.